release_build_files/readme.md

```diff
@@ -634,7 +634,7 @@ workflow use only during the development of your app, not for publicly shipping
 code.
 
 ## Release Notes
-### Upcoming
+### 9.6.0
 -   Changes
```

Comment:
Can you add similar lines for the iOS and Android dependencies like I had done for 9.5.0